### DISCUSSION TITLE
How to configure skip record in CsvHelper.Configuration if all non-optional fields are empty

### DISCUSSION THREAD

**Discussion by DreamThemeGH** [JoshClose/CsvHelper#1882] [Q&A]
Hi!

For a long time now I have not been able to find a solution to how to configure skipping of rows if all required (not optional) columns are empty.
Help me please. I don't know how to get the option flag for a column in a RAW array of fields to check if they are all empty. It is difficult to do this when the order of the columns can be changed and mapping by name is used.

If there is a bool flag for skipping lines, if all non-optional fields are empty, then this would be enough for my case.

Examples and details on the [forum](https://stackoverflow.com/questions/69544017/how-to-configure-skip-record-in-csvhelper-configuration-if-all-non-optional-fiel)

**Comment by JoshClose** [comment]
I don't see a good way of doing this. The `IReaderRow` should probably be passed in to `ShouldSkipRecord` instead of a `string[]`.

**Reply by JoshClose** [reply]
I created a feature request for this. https://github.com/JoshClose/CsvHelper/issues/1883

**Reply by DreamThemeGH** [reply]
Thanks
